### PR TITLE
Check file parts are less than 100MB

### PR DESCRIPTION
**Pull request by edpaget**
Resolves department-of-veterans-affairs/vets-contrib#209

We agreed with the Central Mail team that there shouldn't be limits on the size of the entire package of files uploaded to the, but that each form or attachment should be limited to 100MB.

### CODE REVIEW

**Comment by edpaget** [comment]
Nginx config changes here: https://github.com/department-of-veterans-affairs/devops/pull/3012

File: modules/vba_documents/app/workers/vba_documents/upload_processor.rb [hunk 1]
```diff
@@ -154,19 +155,28 @@ def perfect_metadata(parts, upload, timestamp)
...
         metadata["ahash#{i + 1}"] = att_info[:hash]
         metadata["numberPages#{i + 1}"] = att_info[:pages]
       end
       metadata
     end
 
+    def check_size(file_path)
+      if File.size(file_path) > MAX_PART_SIZE
+        raise VBADocuments::UploadError.new(code: 'DOC106',
+                                            detail: 'Max part size exceeded. Limit is 100MB.')
```
Comment (patrickvinograd) [inline modules/vba_documents/app/workers/vba_documents/upload_processor.rb]: Can you revise text slightly: 'Maximum document size exceeded. Limit is 100MB per document.'

File: modules/vba_documents/app/workers/vba_documents/upload_processor.rb [hunk 2]
```diff
@@ -14,6 +14,7 @@ class UploadProcessor
     SUBMIT_DOC_PART_NAME = 'document'
     REQUIRED_KEYS = %w[veteranFirstName veteranLastName fileNumber zipCode].freeze
     FILE_NUMBER_REGEX = /^\d{8,9}$/
+    MAX_PART_SIZE = 100_000_000 # 100MB
```
Comment (patrickvinograd) [inline modules/vba_documents/app/workers/vba_documents/upload_processor.rb]: Hmm, I wonder if the limit is really 100 MB or 100 MiB. This is more conservative, so 👍

Comment (edpaget) [reply modules/vba_documents/app/workers/vba_documents/upload_processor.rb]: I figured it was less likely to surprise us this way.

**Pull request merged**